pkgs/tools/system/testdisk/default.nix

```diff
@@ -0,0 +1,71 @@
+{ mkDerivation
+, stdenv
+, fetchurl
+, ncurses
+, libuuid
+, pkgconfig
+, enableJpeg ? false, libjpeg ? null
+, enableZlib ? false, zlib ? null
+, enableNtfs ? false, ntfs3g ? null
+, enableExtFs ? false, e2fsprogs ? null
+, enableEwf ? false, libewf ? null
+, enableQt ? false, qtbase ? null, qttools ? null, qwt ? null
+}:
+
+assert enableJpeg -> libjpeg != null;
+assert enableZlib -> zlib != null;
+assert enableNtfs -> ntfs3g != null;
+assert enableExtFs -> e2fsprogs != null;
+assert enableEwf -> libewf != null;
+assert enableQt -> qtbase != null;
+assert enableQt -> qttools != null;
+assert enableQt -> qwt != null;
+
+mkDerivation rec {
```

Comment:
This brings in a qt dependency even if `enableQt = false`. It shoud be `(if enableQt then mkDerivation else stdenv.mkDerivation)`, I think